packages/edit-navigation/src/components/inspector-additions/manage-locations.js

```diff
@@ -0,0 +1,158 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { useContext } from '@wordpress/element';
+import {
+	Spinner,
+	SelectControl,
+	CheckboxControl,
+	Button,
+	Modal,
+} from '@wordpress/components';
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { useMenuLocations, MenuIdContext } from '../../hooks';
+
+export default function ManageLocations( {
+	onSelectMenu,
+	isModalOpen,
+	openModal,
+	closeModal,
+} ) {
+	const menus = useSelect( ( select ) => select( 'core' ).getMenus(), [] );
```

Comment:
This was removed in https://github.com/WordPress/gutenberg/pull/29906, so will also need to bring this change over here.